### ISSUE TITLE
Is there a possibility to port it to fabric?

### ISSUE THREAD

**Issue by sMouaad** [earomc/chestlocker#2]
that would be very neat!

**Comment by earomc** [comment]
I agree! Do you mean as server-side mod only? Or also client-side? Because with a client-side mod we would have more options making the mod a more immersive experience.

However as of right now, me making this a mod seems unlikely because I have withdrawn more and more from Minecraft programming and this would be quite an undertaking.
